lib/netstd/README.md

```diff
@@ -0,0 +1,54 @@
+# Apache Thrift netstd
+
+Thrift client library for Microsoft .NET Standard 
```

Comment:
Perhaps additional information on why one would use netstd instead of netcore or csharp (they are deprecated)?